src/python/WMComponent/JobSubmitter/JobSubmitterPoller.py

```diff
@@ -728,6 +778,8 @@ def algorithm(self, parameters=None):
             if self.useReqMgrForCompletionCheck:
                 # only runs when reqmgr is used (not Tier0)
                 self.removeAbortedForceCompletedWorkflowFromCache()
+                agentConfig = self.reqAuxDB.getWMAgentConfig(self.config.Agent.hostName)
+                self.condorOverflowFraction = agentConfig.get("CondorOverflowFraction", 0.2)
```

Comment:
Since we already get the aux db config in L815. How about we move this 
`self.condorOverflowFraction = agentConfig.get("CondorOverflowFraction", 0.2)`
to L817?

Reply:
Actually L815 need to be moved here 781 otherwise Tier0 agent will crash.